packages/tools/fluid-runner/api-extractor-lint.json

```diff
@@ -1,13 +1,4 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "../../../common/build/build-common/api-extractor-lint.json",
-	"messages": {
```

Comment:
The base rules have included this as an error for a while now. Updating this file must have been missed, or it was added after.